ansible-ipi-install/roles/installer/tasks/25c_disconnected_registry_create.yml

```diff
@@ -0,0 +1,186 @@
+---
+
+- name: Create a disconnected registry
+  block:
+    - name: Make sure needed packages are installed
+      package:
+        state: present
+        name: "{{ item }}"
+      loop:
+        - "{{ packages_registry | default([]) }}"
+      become: true
+    
+    - name: open firewall port {{ registry_port }}
+      firewalld:
+        port: "{{ registry_port }}/tcp"
+        permanent: yes
+        immediate: yes
+        state: enabled
+      become: true
+    
+    - name: query getent for homedir
+      getent:
+        database: passwd
+        key: "{{ ansible_ssh_user }}"
+    
+    - name: set homedir fact
+      set_fact:
+        homedir: "{{ getent_passwd | json_query('*[4]') | first }}"
```

Comment:
Not sure you actually need to do this. You should be able to use 
```
{{ ansible_env.HOME }}
```

Reply:
done will be included in next commit.